src/airflow/dags/airnow_bam_data.py

```diff
@@ -92,10 +92,11 @@ def send_to_bigquery(data: pd.DataFrame):
     def send_measurements_to_api(data: pd.DataFrame):
         from airqo_etl_utils.airqo_api import AirQoApi
         from airqo_etl_utils.airqo_utils import AirQoDataUtils
+        from airqo_etl_utils.constants import Tenant
 
         restructured_data = AirQoDataUtils.process_airnow_data_for_api(data=data)
         airqo_api = AirQoApi()
-        airqo_api.save_events(measurements=restructured_data, tenant="airqo")
+        airqo_api.save_events(measurements=restructured_data, tenant=Tenant.US_EMBASSY)
 
```

Comment:
tenant is still USEmbassy here.... @NoahNsimbe

Reply:
This has not effect. it is used to identify the network. @Baalmart.